model-engine/model_engine_server/common/dtos/llms.py

```diff
@@ -528,6 +528,10 @@ class CreateBatchCompletionsRequest(BaseModel):
     """
     Maximum runtime of the batch inference in seconds. Default to one day.
     """
+    max_gpu_memory_utilization: Optional[float] = Field(default=0.9, le=1.0)
```

Comment:
consider removing this. we have been trying to make batch completion params as simple as possible so users don't need to specify things like gpu/cpu count

Reply:
Created a separate DTO for engine request